data-raw/revised_snorkel_db_pull.R

```diff
@@ -0,0 +1,229 @@
+                                    TRUE ~ instream_cover),
+         substrate = ifelse(is.na(substrate), NA, str_arrange(substrate)),
+         substrate = as.numeric(case_when(substrate == "2344" ~ "234",
+                               substrate == "350" ~ "35",
+                               TRUE ~ substrate)),
+         unit = case_when(unit == "32A" ~ "32", # cleaning up these units because they are not in the snorkel_section_river_miles table
+                                 unit == "329.5" ~ "329",
+                                 unit == "255A" ~ "255",
+                                 unit == "266A" ~ "266",
+                                 unit == "172B" ~ "172",
+                                 unit == "26A" ~ "26",
+                                 unit == "329B" ~ "329",
+                                 unit == "111A" ~ "111",
+                                 unit %in% c("274B", "274A") ~ "274",
+                                 unit == "448A" ~ "448",
+                                 unit == "271B" ~ "271",
+                                 unit %in% c("272A", "272B") ~ "272",
+                                 unit == "273B" ~ "273",
+                                 unit == "118A" ~ "118",
+                                 unit == "335B" ~ "335",
+                                 unit == "487B" ~ "487",
+                                 TRUE  ~ unit)) |>  glimpse()
+
+# Clean snorkel survey metadata
+# TODO are mos and hatchery ditches the same thing? There is no slide for Mos riffle but it is on the map, making mos hatchery for now
+# TODO existing units to clean are located in messy units object below, would be good to ask Casey
+cleaner_snorkel_survey_metadata <- raw_snorkel_survey_metadata |>
+  janitor::clean_names() |>
+  select(-snorkelers, -comments, -shore_crew, -time_of_temperature, -snorkel_start_ttime, -snorkel_end_time) |> # removing times because they are just dates(something lost in pull)
+  rename(weather = weather_code) |>
+  mutate(weather = case_when(weather %in% c("CLD", "CLDY") ~ "cloudy",
+                             weather %in% c("CLR (Hot)", "CLR/Hot", "Hot and CLR", "CLR Hot") ~ "clear and hot",
+                             weather %in% c("RAIN", "RAN", "CLD/RAIN", "LT RAIN", "CLD, Wind, Light Sprinkles") ~
+                               "precipitation",
+                             weather %in% c("CLR 95", "CLR") ~ "clear",
+                             weather %in% c("PT. CLDY", "CLR/CLD") ~ "partly cloudy",
+                             weather %in% c("sun", "SUN") ~ "sunny",
+                             weather == c("CLR WINDY") ~ "clear and windy",
+                             weather == c("WND") ~ "windy",
+                             weather == c("LT CLD/HAZE") ~ "hazy"),
+         section_name = format_site_name(section_name),
+         section_name = case_when(section_name %in% c("Vance W", "Vance West") ~ "Vance Riffle",
```

Comment:
Thanks for going through all of this - its a lot of work. I did something similar for the mini snorkel and thinking it may be good for us to compare the lists. The sites should be pretty similar.